src/query_ctrl.js

```diff
@@ -144,6 +160,15 @@ export class MQEQueryCtrl extends QueryCtrl {
     });
   }
 
+  getFunctions() {
+    var _this10 = this;
+
+    return this.exploreMetrics('functions').then(function (functions) {
```

Comment:
This plugin use ES6, so you can use arrow function to make it simpler:
```
return this.exploreMetrics('functions').then((functions) => {
```
By using arrow functions you don't need to reassign `this`, because arrow function hasn't its own `this`.
So you can write this:
```
getFunctions() {
  return this.exploreMetrics('functions').then((functions) => {
    var segments = this.transformToSegments(functions, true);
    segments.splice(0, 0, angular.copy(this.removeSegment));
    return segments;
  });
}
```